src/app/teacher-round.service.ts

```diff
@@ -42,29 +50,27 @@ export class TeacherRoundService {
    * Create a new round within the current session, mark it as the currently
    * active round, set its initial state, and hook up the TimerService so that
    * when the timer ticks, the round updates.
-   *
-   * TODO: As of this iteration, this function just re-uses a single demo
-   * round and re-populates it with data, rather than creating a new round.
-   * We also don't bother with marking it as the current round.
    */
   // In the future, we might get sessionId from a TeacherSessionService, rather
   // than passing it in as a parameter.
-  startNewRound(sessionId: string, roundData: FirebaseRound): void {
-    // Eventually, we'll create a new round, but for the moment, we'll just use
-    // this one.
-    this.roundPath$.next(demoRoundPath);
-    this.firebaseService.setRoundData(demoRoundPath, roundData);
-  }
+  startNewRound(roundData: FirebaseRound): void {
+    let round: Promise<RoundPath>;
```

Comment:
Why are we creating a variable for this when we never use it to return the promise?